src/seeds.accounts.cpp

```diff
@@ -303,17 +303,14 @@ void accounts::vouchreward(name account) {
 
   // For adding community building points
   bool is_citizen = status.value == name("citizen").value;
```

Comment:
It is nothing maybe, but I think the variable is_citizen is not being used.

Reply:
Correct

Reply:
Removed.